internal/pkg/template/templates/environment/partials/cdn-resources.yml

```
@@ -0,0 +1,24 @@
+cloudfrontdistribution:
```

nit: to be consistent with our other logical IDs
```suggestion
CloudFrontDistribution:
```